internal/web3ext/web3ext.go

```diff
@@ -865,6 +865,12 @@ web3._extend({
 			call: 'scroll_getBlockTraceByNumberOrHash',
 			params: 1
 		}),
+		new web3._extend.Method({
+			name: 'getBlockTraceForCallAt',
+			call: 'scroll_getBlockTraceForCallAt',
+			params: 3,
+			inputFormatter: [null, null, null]
```

Comment:
```suggestion
			params: 3
```

Reply:
Here I just follow
```
		new web3._extend.Method({
			name: 'traceCall',
			call: 'debug_traceCall',
			params: 3,
			inputFormatter: [null, null, null]
		}),
```

So I think it's unnecessary to delete this line.

Reply:
Oh I see.